app/src/CoreGameLogic/Feature/Spielzug/State/PlayerState.php

```diff
@@ -207,6 +214,57 @@ public static function getLastMinijobForPlayer(GameEvents $stream, PlayerId $pla
         return CardFinder::getInstance()->getCardById($minijobWasDoneEvent->minijobCardId, MinijobCardDefinition::class);
     }
 
+    /**
+     * @param GameEvents $stream
+     * @param PlayerId $playerId
+     * @return WeiterbildungCardDefinition|null
+     */
+    public static function getLastWeiterbildungCardDefinitionForPlayer(GameEvents $stream, PlayerId $playerId): ?WeiterbildungCardDefinition
+    {
+        $weiterbildungEvent = self::getLastWeiterbildungsEventForPlayer($stream, $playerId);
+
+        if ($weiterbildungEvent === null) {
+            return null;
+        }
+
+        $cardId = $weiterbildungEvent->weiterbildungCardId;
+
+        /** @var WeiterbildungCardDefinition $cardDefinition */
+        $cardDefinition = CardFinder::getInstance()->getCardById($cardId);
+        return $cardDefinition;
```

Comment:
Funktioniert auch das?
```suggestion
        return CardFinder::getInstance()->getCardById($cardId, WeiterbildungCardDefinition::class);
```

Reply:
sure, is auch paar zeilen weiter oben genau so geschrieben für mini jobs...